### DISCUSSION TITLE
Cannot choose existing business from Stripe Connect

### DISCUSSION THREAD

**Discussion by surjithctly** [community/community#17267] [Other Feature Feedback, Questions, & Ideas]
Hello,

Today I got access to Sponsors, I filled all required steps. Then it asked me to connect stripe account, where I cannot choose any existing. I have used Stripe Connect before and it worked well. But for Github Sponsors, businesses were not shown. Only one showing which says "No name exist", then an option to add one. 

How do I choose an existing business? I'm from India.

**Comment by jlord** [comment]
Hiya! Our Support folks can help you. If you [go to this page](https://support.github.com/contact/account?tags=hubberfy_account) and then select GitHub Sponsors as what you need help with we'll be able to help you sort out your account there ✨
